controllers/chat/group/groupAddMembers.js

```diff
@@ -57,6 +57,13 @@ const groupAddMembers = async (req, res) => {
   try {
     const responseChannel = await channelToAdd?.addMembers(filteredMembersObject);
 
+    await channelToAdd.sendMessage({
+      text: 'Added new members',
+      type: 'system',
+      user_id: userId,
+      members
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code sends a system message whenever new members are added to the channel. However, it's important to consider error handling for this operation. If sending the message fails for any reason (network issues, service unavailability, etc.), the function will throw an exception and stop execution. This could potentially leave your application in an inconsistent state if there are other operations that need to be performed after this.

Consider wrapping the `sendMessage` call in a try-catch block to handle potential errors gracefully. Here's how you might do it:

```diff
   try {
     const responseChannel = await channelToAdd?.addMembers(filteredMembersObject);
 
+    try {
       await channelToAdd.sendMessage({
         text: 'Added new members',
         type: 'system',
         user_id: userId,
         members
       });
+    } catch (error) {
+      console.error('Failed to send system message: ', error);
+    }
 
     try {
       const {betterChannelMember, betterChannelMemberObject, newChannelName, updatedChannel} =
         await BetterSocialCore.chat.updateBetterChannelMembers(
```
This way, even if sending the system message fails, the function can continue executing the remaining operations.


<!-- This is an auto-generated comment by OSS CodeRabbit -->